projects/packages/calypsoify/src/class-jetpack-calypsoify.php

```diff
@@ -33,7 +34,7 @@ class Jetpack_Calypsoify {
 	/**
 	 * Jetpack_Calypsoify constructor.
 	 */
-	private function __construct() {
+	public function __construct() {
```

Comment:
Since we are still using the Singleton pattern to load this class, why are we changing this to public?

Also there's a `Jetpack_Calypsoify::get_instance();` in line `221`. Is this intended?

Reply:
> Since we are still using the Singleton pattern to load this class, why are we changing this to public?

No good reason that I can think of just now, I may have thought it necessary when initially moving code over. Removed now.

> Also there's a Jetpack_Calypsoify::get_instance(); in line 221. Is this intended?

Ah good catch. Since we're calling `get_instance` in both the mu-wpcom package and the Jetpack module directly, that'll be a no. Removed now.